Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -56,7 +56,13 @@ ErrorOr<void> VirtualFileSystem::mount(FileSystem& fs, Custody& mount_point, int
             fs.class_name(),
             inode.identifier(),
             flags);
-        // FIXME: check that this is not already a mount point
+        const auto it = mounts.find_if([&mount_point](Mount const &existing_mount) {
+            return existing_mount.host() && existing_mount.host()->identifier() == mount_point.inode().identifier();
+        });
+        if (it != mounts.end()) {
```

Comment:
nit: There is `AK::any_of(..)` that encapsulates this pattern a bit.

Reply:
Thanks for the tip :)